### PR TITLE
test new helpers once again

### PR DESCRIPTION
**Pull request by maurolepore**

### CODE REVIEW

File: tests/testthat/test-to_title.R [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  data <- tribble(
+           ~input,     ~output,
+      "projected", "Portfolio",
+        "target_",  "Scenario",
+  "anything else", "Benchmark",
+      # Edge cases
+      "Projected", "Benchmark",
+       ".target_", "Benchmark",
+            "xyz", "Benchmark"
+  )
```
Comment (MonikaFu) [inline tests/testthat/test-to_title.R:40]: The above look problematic. I think that the implementation might need to be improved for the next release.

Comment (maurolepore) [reply tests/testthat/test-to_title.R:40]: #439
